modules/roles_profiles/manifests/profiles/disable_uac.pp

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class roles_profiles::profiles::disable_uac {
```

Comment:
Let's move this to `roles_profiles::profiles::suppress_dialog_boxes`  and put it under a Win 11 specific case statement since it has not been tested on the other worker pools.